CMakeLists.txt

```diff
@@ -71,25 +71,27 @@ if (WIN32)
 elseif(UNIX)
     set(PLATFORM "SDL2")
     add_compile_definitions(SDL2=1)
+    
+    # Common compiler options and warning level for CLANG and GCC:
+    add_compile_options(
+        -Wall
+
+        -Wno-unused-variable
+        -Wno-unused-function
+        -Wno-unused-but-set-variable
+        -Wno-sign-compare
+        -Wno-switch
```

Comment:
I'd prefer having -Wswitch because it helps spotting when a case was missed. But I guess one can always manually activate it when working on unfamiliar code...

Reply:
I can enable that